src/assets/abilities.ts

```diff
@@ -77,6 +77,27 @@ export default new GraphQLCollection<string, Pokemon.Ability>([
       name: 'Aroma Veil'
     }
   ],
+  [
+    'asone',
+    {
+      shortDesc: 'The combination of Unnerve and Chilling/Grim Neigh.',
+      name: 'As One'
+    }
+  ],
```

Comment:
```suggestion
```

As One as a standalone ability doesn't exist, only the specifications of it exist.